packages/breadboard-ui/src/elements/input/input.ts

```diff
@@ -295,8 +305,23 @@ export class Input extends LitElement {
       } else {
         const input = form[key];
         if (input && input.value) {
-          const parsedValue = parseValue(property.type, input);
-          data[key] = parsedValue;
+          try {
+            const parsedValue = parseValue(property.type, input);
+            data[key] = parsedValue;
+            //throw new Error("Error when parsing input values.");
+          } catch (error) {
+            if (error instanceof Error) {
```

Comment:
If we get something here that isn't error this will now silently fail. That said I can't immediately think of why we would get anything here that isn't an instance of `Error` (which makes me wonder about the `instanceof`, but maybe I'm missing something? I like us emitting a more "controlled" Event here, though.

Reply:
This was amended in the new commit by using the new InputErrorEvent type and removing the `instanceof`.